Apps/Playground/UWP/App.cpp

```diff
@@ -112,9 +112,16 @@ void App::Run()
 // class is torn down while the app is in the foreground.
 void App::Uninitialize()
 {
-    m_inputBuffer.reset();
-    m_runtime.reset();
-    Babylon::DeinitializeGraphics();
+    if (m_inputBuffer)
```

Comment:
why do you need these guards?

Reply:
If I call `Babylon::DeinitializeGraphics` when `Babylon::InitializeGraphics` has not been called, bgfx dies. So I'm just using `m_runtime` as a flag to know whether to do this since as the code currently stands they are always initialized/deinitialized together.

Reply:
ok for now. i suspect we will change this behavior soon anyways.